src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -6162,6 +6163,7 @@ export interface NS {
    *
    * Write data to the given Netscript port.
    * @param portNumber - Port to write to. Must be a positive integer.
+   * @param data - the data to write to the port
```

Comment:
No caps, no period? Another line uses `Data to write.`, if that's not an OK standard, should probably decide one.